README.md

```diff
@@ -132,31 +81,67 @@ output = scenario.simulate(object_path="vehicle.obj",
                            resolution = "medium")
 ```
 
-The user needs to specify the object to insert inside the wind tunnel and thereafter,
-select the total simulation time, the step between outputs and the resolution of the simulation.
+The user must provide the path for the object to be inserted in the wind tunnel. Additionally, users can choose the total simulation time, the time step interval to output data and the resolution of the simulation.
 
-After the simulation has finished, the user has the possibility to obtain several metrics:
+After the simulation has finished, the user can obtain several metrics and visuals. An example here is given for a slice of the flow:
 
 ```python
 
-pressure_field = output.get_physical_field("pressure")
+flow_slice = output.get_flow_slice(simulation_time=100,
+                                   plane="xz")
+flow_slice.render("velocity")
+```
+
+![F1 car in WindTunnel](resources/media/wind_tunnel.png)
+
+### Fluid tank
+
+This scenario simulates the motion of a fluid in a cubic or cylindrical tank.
+Fluid is injected into the tank via an inlet located at the top of the tank and
+flows out of the tank via an outlet located at the bottom of the tank. The
+motion of the fluid is controlled by gravity. The simulation is performed using
+the [Smoothed Particle Hydrodynamics](https://en.wikipedia.org/wiki/Smoothed-particle_hydrodynamics)
+method.
+
+#### Example
+
+Initialize the scenario:
+
+```python
+from inductiva import fluids
+scenario = fluids.scenarios.FluidTank(shape=fluids.shapes.Cube(),
```

Comment:
You need to add `from . import shapes` to the file `inductiva/fluids/__init__.py` for this access to the `Cube` class to work.

Reply:
I think it is the main point of this scenario, so I will make this addition!